### PR TITLE
0.1-pre.1 releases

### PR DESCRIPTION
**Pull request by franziskuskiefer**
Here some fixes for the releases.
We need to figure out a good name for the main package. Essentially, it should be renamed to `cargo-hax` unless we want something else.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,13 @@ Note:
  * The subcommand `cargo hax into` takes also options, list them with `cargo hax into --help`.
 
 ## Installation
+
+Install the latest release from crates.io with
+
+```bash
+cargo +nightly-2023-06-02 install hax-subcommands
+```
+
```
[the remark this answers is not in the document]
Comment (franziskuskiefer) [reply README.md:35]: Yep, this only installs the Rust pieces. I just wanted to dump it somewhere. I agree that it's not great that it doesn't fully work yet ... But we should also rename the package. So this isn't final in any case.
If we can agree on a name, we can move this somewhere else temporary and then get something done together with #261 .

[the remark this answers is not in the document]
Comment (W95Psp) [reply README.md:35]: So you mean renaming the crate `hax-subcomands` (located in dir `cli/subcommands`) into `cargo-hax`?
That seems fine to me!

I think renaming the crate & moving this paragraph out of the README for now until we close #259 is a good strategy!

[the remark this answers is not in the document]
Comment (W95Psp) [reply README.md:35]: (added some comment in #259 relatively to the README)

**Review by W95Psp — APPROVED** [review]
LGTM :)

**Pull request merged**